DependencyInjection/Compiler/MailerCompilerPass.php

```diff
@@ -19,8 +18,19 @@ class MailerCompilerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        $this->registerOutboundMessageTransformers($container, $container->getDefinition('everlution.email.outbound.synchronous_mailer'));
-        $this->registerOutboundMessageTransformers($container, $container->getDefinition('everlution.email.outbound.asynchronous_mailer'));
+        if ($container->hasDefinition('everlution.email.outbound.synchronous_mailer')) {
```

Comment:
Why this conditions? If the services are not defined, then app should throw an error.